TASVideos/TagHelpers/WikiMarkupTagHelper.cs

```diff
@@ -94,10 +96,22 @@ bool IWriterHelper.CheckCondition(string condition)
 			return null;
 		}
 
+		private static readonly IDictionary<string, Type> ViewComponents = Assembly

```

Comment:
Should this code, or some code somewhere, verify that each and every `public const string Foo = "bar"` actually has an implementer?  As it stands now, `WikiModules.IsModule` will return true if the field is defined, but that's no guarantee that any particular class has the attribute on it for that value.